src/Processors/QueryPlan/ReadFromMergeTree.cpp

```diff
@@ -2107,6 +2114,7 @@ ReadFromMergeTree::AnalysisResultPtr ReadFromMergeTree::selectRangesToRead(
             indexes->key_condition,
```

Comment:
we are forwarding so many members from the `indexes` variable, instead can we use `indexes` directly?